solr/solr-ref-guide/ui-src/partials/head-icons.hbs

```diff
@@ -1,3 +1,20 @@
 <!-- Solr Changes - Start -->
     <link rel="icon" href="{{{uiRootPath}}}/img/favicon.ico" type="image/x-icon">
+    {{#if env.MATOMO_ENABLED}}
+    <script>
```

Comment:
I piggy-backed on `head-icons.hbs` instead of customizing `head.hbs`. See https://gitlab.com/antora/antora-ui-default/-/blob/master/src/partials/head.hbs for the default template. I could of course fork `head.hbs`, add a new `head-matomo.hbs` to the end to separate it more. But the `head-icons` script is already the last in <head> and already customized, so an easier choice. I can be persuaded otherwise though.

Reply:
So I did some more research, and turns out the [head-scripts.hbs](https://gitlab.com/antora/antora-ui-default/-/blob/master/src/partials/head-scripts.hbs) template only contained some google-analytics stuff that we don't want anyway, so I moved our matomo script there instead.